src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+    private String color;
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+

```

Comment:
would be more idiomatic to create a constructor with color in this case, since enforce setting color anyway for each subclasss

Reply:
this comment is not clear, please explain